### ISSUE TITLE
Pikel's minions don't disappear when he dies on a different floor

### ISSUE THREAD

**Issue by powerbf** [crawl/crawl#1861]
webtiles trunk 0.27-a0-830-ga1c911b

In my current game on CPO, I separated Pikel from some of his minions. When I killed him, the minions that were on the same floor as us disappeared, but the ones that were on a different floor survived.

**Comment by ebering** [comment]
This has also been reported on mantis: https://crawl.develz.org/mantis/view.php?id=12541

Can you give any more information about what you did between killing Pikel and going back to the floor with the minions?

**Comment by powerbf** [comment]
After killing Pikel, I just rested until I was healed and then headed downstairs. Got a bit of a shock when 2 or 3 lemures were hanging around the stairs.

**Comment by RojjaCebolla** [comment]
Did you have Sacrifice Love from Ru?

**Comment by rawlins** [comment]
I did spend some time trying to replicate this the other day but couldn't get anywhere, so there's some factor over and above normal movement across levels. I'm 85% sure this isn't a dactions issue though. One useful note in dev chat from hellmonk is:

```is there a way that band status could be getting reset somehow
the dact is checking that the monster is a lemure, that it's got a band member flag, and that it's got "pikel_band" monster property
idk if anything can reset those latter things```

**Comment by ebering** [comment]
Possibly related to #1983 since that is definitely a dactions issue and there's also a ticket with Yred.

**Comment by DracoOmega** [comment]
Should be fixed by 058985b245d8e15b62c93770fc0a68c0a486c9b2
